### DISCUSSION TITLE
Getting error while creating project

### DISCUSSION THREAD

**Discussion by Shubham1406220** [argoproj/argo-cd#12969] [General]
argocd proj create dev --file project-dev.yaml

Getting this error while running the command from cli
`FATA[0000] error reading proj from uri: %!w(<nil>) `

**Comment by rahulchak** [comment]
I am also getting the same error, I have the same command worked in different projects before, I cannot find out if its something related to version. This is the version I am using - argocd: v2.6.7+5bcd846, running on Debian 11 and the exact error below - 
FATA[0000] error reading proj from uri: %!w(<nil>)

**Comment by alexmt** [comment]
It is definitely a bug. Thanks for bringing this up! Created an issue: https://github.com/argoproj/argo-cd/issues/13129
